Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -45,8 +45,11 @@
 "action-saved-accessibility" = "Saved. Activate to unsave.";
 "action-share" = "Share";
 "action-unsave" = "Unsave";
+"adjust-text-sizing" = "Adjust text sizing";
```

Comment:
@joewalsh Thoughts on whether we should keep adhering to the TWN string naming convention we had before your localization refactors? Now that strings "live" where they're used the verbose naming we had used before is arguably less useful...